base/ircd.yaml

```diff
@@ -450,13 +455,13 @@ opers:
         class: "server-admin"
 
         # custom whois line
-        whois-line: is a cool dude
+        whois-line: is an IRC operator
 
         # custom hostname
-        vhost: "n"
+        vhost: "hashbang.sh"
```

Comment:
We should set this to something like `staff.hashbang.sh` instead. Requesting suggestion from @lrvick @drGrove @daurnimator